public_html/wp-content/mu-plugins/blocks/assets/src/speakers/block-content.js

```diff
@@ -118,18 +150,29 @@ class SpeakersBlockContent extends Component {
 												<a
 													className={ 'wordcamp-speaker-session-link' }
 													href={ session.link }
+													// is there an esc_url() equivalent in g? or do we not need it because we're returning an Element, which gets escaped automatically?
```

Comment:
I'm not aware of a G equivalent of `esc_url`. I'm also not sure if the escaping gets handled by something else or not. I know I saw a similar implementation without escaping in some component in Gutenberg, but that doesn't necessarily mean it's safe ;P